source/gui/addonStoreGui/viewModels/store.py

```diff
@@ -612,6 +619,42 @@ def _getAvailableAddonsInBG(self):
 		core.callLater(delay=0, callable=self.detailsVM.updated.notify, addonDetailsVM=self.detailsVM)
 		log.debug("completed refresh")
 
+	def cancelDownloadAddon(self, listItemVM: AddonListItemVM[_AddonStoreModel]):

```

Comment:
If this is a helper method please make it private

```suggestion
	def _cancelDownloadForAddon(self, listItemVM: AddonListItemVM[_AddonStoreModel]):
```

or 

```suggestion
	def _cancelAddonDownload(self, listItemVM: AddonListItemVM[_AddonStoreModel]):
```